tests-integration/src/test_util.rs

```diff
@@ -126,6 +166,7 @@ fn get_test_store_config(
 enum TempDirGuard {
     File(TempDir),
     S3(TempFolder),
+    Oss(TempFolder),
```

Comment:
Prefer `OSS` which is the same as `ObjectStoreConfig`.

Reply:
`Oss` is fine according to [style-guide](https://pingcap.github.io/style-guide/rust/naming.html), we (me and @evenyag) had a discussion about this on https://github.com/GreptimeTeam/greptimedb/pull/850#discussion_r1064320339

Reply:
It's okay, but it's better to keep it consistent between `TempDirGuard` and `ObjectStoreConfig`.

Reply:
Renamed OSS to Oss for `ObjectStoreConfig`